src/Compilers/CSharp/Portable/Binder/Binder_Attributes.cs

```diff
@@ -438,7 +438,7 @@ private TypeSymbol BindNamedAttributeArgumentType(AttributeArgumentSyntax namedA
 
             bool invalidNamedArgument = false;
             TypeSymbol namedArgumentType = null;
-            invalidNamedArgument |= (namedArgumentNameSymbol.DeclaredAccessibility != Accessibility.Public);
+            invalidNamedArgument |= namedArgumentNameSymbol.DeclaredAccessibility != Accessibility.Public;
```

Comment:
I like the parens here ... I don't have to remember relative precedence.

Reply:
Yes.  this is an interesting case since this is affectively "x = x | (y != z)".  I *think* the rule should be that if you have simple assignment, it's very redundant and unnecessary to have parens.  i.e.:

```x = y != z``` is clearly equivalent (to most people i hope) to  ```x = (y != z)``` not, somehow ```(x = y) != z```.

However, for *compound* assignment, there could be an argument that this is less clear, and the presence of the operator in ```|=``` means that we should not consider the parens unnecessary for readability.

In a sense, these are just another form of binary operators.  They always have the lowest precedence, but someone could still want that precedence for clarity purposes.